### PR TITLE
Styles+profile

### PR DESCRIPTION
**Pull request by adrnchk**
image
image
This is how it looks like, but still work is in progress...

### CODE REVIEW

**Comment by adrnchk** [comment]
image
profile page wip

**Comment by adrnchk** [comment]
image

File: DogeNet.BLL/Features/Account/UpdateAccount/UpdateAccountHandler.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        {
+            this.context = context;
+            this.mapper = mapper;
+        }
+
+        public async Task<AccountDetailsModel> Handle(UpdateAccountCommand request, CancellationToken cancellationToken)
+        {
+            var user = await this.context.AppUsers.FindAsync(request.model.Id);
+
+            user.FirstName = request.model.FirstName;
```
Comment (oleh-leobit) [inline DogeNet.BLL/Features/Account/UpdateAccount/UpdateAccountHandler.cs:34]: Mapper?

File: DogeNetWebClient/src/app/shared/components/nav-bar/nav-bar.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-nav-bar',
+  templateUrl: './nav-bar.component.html',
+  styleUrls: ['./nav-bar.component.css']
+})
+export class NavBarComponent implements OnInit {
+
+  constructor() { }
```
Comment (oleh-leobit) [inline DogeNetWebClient/src/app/shared/components/nav-bar/nav-bar.component.ts:10]: Could it be just an empty component?

**Review by oleh-leobit — APPROVED** [review]
There are a few minor comments, but you are good to merge. I like designs!

**Pull request merged**